examples/text-utilities/src/text-utilities.js

```diff
@@ -52,8 +52,12 @@ function addBaselines(layer, fragments) {
     const shape = new sketch.Shape({
       parent: group,
       frame: rect,
-      fills: ['#ff000090'],
-      borders: [],
+      style: {
```

Comment:
not sure if this change is needed, if a fill is a string, it will assume that it is a `FillType.color`

Reply:
Yes, you're right! But it is still necessary to wrap inside `style` object

Reply:
Indeed. But `sketch.Style.FillType.color` is a Number and should stay a number. You can either use `sketch.Style.FillType.color` or `'color'`

Reply:
Yes, it's a miss. Maybe you should also mention it in document, because the fills type in document is `Object[]`